engine/src/faction_generic.cpp

```diff
@@ -1,14 +1,18 @@
 #include "faction_generic.h"
 #include "cmd/unit_generic.h"
+#include "string.h"
+
+
 
 using namespace FactionUtil;
 
 vector< boost::shared_ptr<Faction> >factions;  //the factions
 
 void Faction::ParseAllAllies()
 {
-    //MSVC has a bug where it won't let you reuse variables intitliezed in the paramater list of the for loop
+    //MSVC has a bug where it won't let you reuse variables initialized in the paramater list of the for loop
```

Comment:
I'm pretty sure MSVC doesn't do that anymore. 😄

Reply:
who knows, we'll see once we figure out the MSVC compilation